pkg/util/commons.go

```diff
@@ -1075,3 +1076,18 @@ func YesNo(b bool) string {
 	}
 	return "No"
 }
+
+// Return true or false according to where medication detail matches
+func IsSameMedicationName(med1, med2, st1, st2 string) bool {
```

Comment:
What is the usecase of this? NDC will be always unique for each drug, matching with name is not needed at all.

Reply:
@sarweshmaharjan could you please share the usecase here?